collector/ethtool_linux.go

```diff
@@ -446,7 +446,7 @@ func (c *ethtoolCollector) Update(ch chan<- prometheus.Metric) error {
 			}
 		}
 
-		if stats == nil || len(stats) < 1 {
+		if len(stats) < 1 {
```

Comment:
I have to admit, I find it a little bit strange to test for a zero length like this, considering that it's impossible for `len(foo)` to be negative (or fractional). Why not simply `len(stats) == 0`?

Reply:
Yea, that's probably better.